### DISCUSSION TITLE
[Question] 有多项选择的组件吗

### DISCUSSION THREAD

**Discussion by zhang-xiaoyan** [opensumi/core#3323] [General]
**描述你的问题（Describe you question here）**
请问：quickPickService 或者其他服务，是否提供了内置的可以多项选择的组件
类似这样的组件，每次可以多项选择的组件
![8e32df80df57c346a87ade3d2030cb5](https://github.com/opensumi/core/assets/16955480/f430b871-2334-46ad-b365-2f0874486f4c)

**Comment by bytemain** [comment]
有一个 canPickMany 的入参：

![CleanShot 2024-01-30 at 10 24 23@2x](https://github.com/opensumi/core/assets/13938334/99cd0309-1e76-4d4a-aef8-599dcbd831e9)

```ts
this.quickPick.show(['a', 'b', 'c'], { placeholder: 'Select Environment', canPickMany: true });
```

**Reply by zhang-xiaoyan** [reply]
麻烦问一下 
1、确定按钮点击无反应，我该怎么处理呢
2、每一个 item 的底色有办法去掉吗
![c6626091abdafb235c6385f69a2ad58](https://github.com/opensumi/core/assets/16955480/8d1ead42-f068-47b9-872f-006b0f5acb1b)

**Reply by bytemain** [reply]
什么版本的 opensumi

**Reply by zhang-xiaoyan** [reply]
2.26.8

**Reply by bytemain** [reply]
我写了个 demo，用了 2.26.8，是可以多选的点击确定的：
- https://github.com/opensumi/opensumi-module-samples/pull/36

![CleanShot 2024-01-30 at 16 43 34@2x](https://github.com/opensumi/core/assets/13938334/9219f9c6-8f61-441f-91aa-5d1b260af6f6)

可以分享下代码吗？

**Reply by zhang-xiaoyan** [reply]
我分享一下我的代码：
1、看了一下，没有发现不一样的地方，之前用过 quickPickService.show 这个方法，没有加  canPickMany 这个参数，是好用的。
2、搜索、全选、单选都没有问题。只有那个确定按钮鼠标滑过没有反应，点击也没有反应。
![1706605723436](https://github.com/opensumi/core/assets/16955480/d4364dd1-6267-49fb-af27-d1e2dfebb2a0)

![image](https://github.com/opensumi/core/assets/16955480/5a0562b7-a7e3-4340-8d90-c42cc907d82c)

**Reply by bytemain** [reply]
![CleanShot 2024-01-30 at 17 25 01@2x](https://github.com/opensumi/core/assets/13938334/307bea6d-7d95-43e0-9329-46775bd0c03b)

确实没有复现点击不了的问题，你可以添加一个参数让这个框不会隐藏：
          ignoreFocusOut: true,

来调试一下

**Reply by zhang-xiaoyan** [reply]
好的，我试试，麻烦问一下，这个【确定】按钮有回调事件吗

**Reply by bytemain** [reply]
是在封装好的内部绑定了回调的

**Reply by zhang-xiaoyan** [reply]
好的，那我再试试，麻烦问一下，有中文文档吗，像 antd 那样的文档，例如都包含哪些组件，组件都包括哪些功能，哪些参数、哪些方法等，我现在找东西，都直接去 node_modules 或者github上。是否有更好的方式去查看呢。
